crds/workspace.devfile.io_devworkspaces.yaml

```diff
@@ -6620,6 +6983,126 @@ spec:
+                                                        description: The remote name
+                                                          should be used as init.
+                                                          Required if there are more
+                                                          than one remote configured
+                                                        type: string
+                                                      revision:
+                                                        description: The revision
+                                                          to checkout from. Should
+                                                          be branch name, tag or commit
+                                                          id. Default branch is used
+                                                          if missing or specified
+                                                          revision is not found.
+                                                        type: string
+                                                    type: object
+                                                  gitLocation:
+                                                    description: Location of the Dockerfile
+                                                      in the Git repository when using
+                                                      git as Dockerfile src.
+                                                    type: string
+                                                  remotes:
+                                                    additionalProperties:
+                                                      type: string
+                                                    description: The remotes map which
+                                                      should be initialized in the
+                                                      git project. Must have at least
+                                                      one remote configured
+                                                    type: object
+                                                type: object
+                                              locationType:
+                                                description: Type of Dockerfile location
+                                                enum:
+                                                - Uri
+                                                - Registry
+                                                - Git
+                                                type: string
+                                              registry:
+                                                description: Dockerfile's Devfile
+                                                  Registry source
+                                                properties:
+                                                  devfileRegistryUrl:
```

Comment:
I believe there is a place for duplication.
First of all, maybe it makes more sense to commit devfile word or move it to the upper level:
```
components:
- image:
    devfileRegistry:
      url/registryUrl: ''
      id: ''
```
Also, we have registry word on both levels, but I'm not sure it's a good idea to get rid of it on lower level.

Reply:
sure sounds good, I'll make this change!

Reply:
updated!